cmd/create.go

```diff
@@ -422,7 +422,7 @@ func createHost(provider, name, region, zone, projectID, userData, inletsPort st
 	} else if provider == "gce" {
 		return &provision.BasicHost{
 			Name:     name,
-			OS:       "projects/debian-cloud/global/images/debian-9-stretch-v20191121",
```

Comment:
I suspect there was a good reason why we used Debian here, but Stretch is now out of support.

@utsavanand2 do you remember why? cc @maelvls Perhaps Debian was just so much quicker to boot up a VM?

Reply:
@alexellis Yes, we tried out both Ubuntu and Stretch and at the time we built this, Stretch was marginally faster on the smallest f1-micro machine type. We might want to try out this minimal image and see how this works out `projects/ubuntu-os-cloud/global/images/ubuntu-minimal-2004-focal-v20210707`

Reply:
If you want, i can change to minimal and give it a spin again

Reply:
@alexellis or @utsavanand2, do you have any rough expected time for booting? So i can compare or should we just build on minimal and if it works its fine?